src/main/java/org/embulk/output/elasticsearch/ElasticsearchHttpClient.java

```diff
@@ -48,6 +48,7 @@
     // @see https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/cluster/metadata/MetaDataCreateIndexService.java#L108
     private final long maxIndexNameBytes = 255;
     private final List<Character> inalidIndexCharaters = Arrays.asList('\\', '/', '*', '?', '"', '<', '>', '|', '#', ' ', ',');
+    private final long maxSnapshotWaitingTime = 1800;
```

Comment:
30 mins sounds reasonable for me. can I ask you that it's good idea to parameterize this field as config option?